practicals/Recommender_Systems.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>"At least no" - "At least now"</p><p> </p><p>Also, I don't know how much you should dive into whether or not phones listen to users, since this is controversial and the answer is complex. You've mentioned it quite a few times, and I don't think it really adds much value to the practical.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>